apps/desktop/chrome.py

```diff
@@ -0,0 +1,21 @@
+
+from selenium import webdriver
+from selenium.webdriver import Chrome
+
+
+class DesktopChrome(object):
+    def __init__(self, proxy, *args):
+        self.proxy = proxy
+
+    def start(self, url="about:blank"):
+        print("START")
```

Comment:
nit: can we remove this print statement?

Reply:
I just noticed we have this in `firefox.py` can you remove it from both?

Reply:
done